modules/javafx.graphics/src/main/java/com/sun/javafx/text/PrismTextLayout.java

```diff
@@ -1396,8 +1396,20 @@ private void layout() {
             int lineStart = line.getStart();
             RectBounds bounds = line.getBounds();
 
+            float contentWidth = bounds.getWidth();
+            if (isMirrored()) {
+                float runWidth = 0;
```

Comment:
I checked if it makes sense to cache runWidth in the PrismTextLine - it does not, so this implementation is ok.